src/utils/templates.ts

```diff
@@ -54,18 +68,17 @@ export async function findConflicts(files: CopyDescriptor[], force: boolean = fa
 export async function copyFiles(files: CopyDescriptor[], options: { [key: string]: any }) {
   return Promise.all(
     files.map(file => {
-      const { sourcePath, targetFolder, fileName } = file;
+      const { sourcePath, fileName } = file;
 
       if (sourcePath instanceof URL) {
-        return copyRemote(sourcePath, targetFolder, fileName);
-      } else {
-        return copyLocal(sourcePath, targetFolder, fileName, options);
+        return copyRemote(sourcePath, fileName);
```

Comment:
Looking at this again, I realize I could have made this function better... Since you edited it, consider changing it to this:
``` ts
export async function copyFiles(files: CopyDescriptor[], options: { [key: string]: any }) {
  return Promise.all(files.map(({ sourcePath, fileName }) => sourcePath instanceof URL ?  copyRemote(sourcePath, fileName) : copyLocal(sourcePath, fileName, options)));
} 
```